### PR TITLE
Update Web3-Onboard Guide

### PR DESCRIPTION
**Pull request by fhildeb**

### CODE REVIEW

File: docs/learn/dapp-developer/web3-onboard.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+sidebar_label: '🛳️ Connect Web3-Onboard'
+sidebar_position: 9
+---
+
+# Universal Profile Integration for Web3-Onboard
+
+The [Web3-Onboard](https://onboard.blocknative.com/) library is an open-source, framework-agnostic JavaScript tool to onboard users to Web3 apps. Developers can integrate it into their dApp to handle the routing for the different extensions and wallets, simultaniously docking onto the Ethereum provider.
```
Comment (CJ42) [inline docs/learn/dapp-developer/web3-onboard.md:8]:
```suggestion
The [Web3-Onboard](https://onboard.blocknative.com/) library is an open-source, framework-agnostic JavaScript tool to onboard users to Web3 apps. Developers can integrate it into their dApp to handle the routing for the different extensions and wallets, simultaneously docking onto the Ethereum provider.
```

File: docs/learn/dapp-developer/web3-onboard.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+![web3-onboard-view](/img/extension/web3-onboard.png)
+
+Our [Web3-Onboard configuration](https://github.com/lukso-network/web3-onboard-config) can be used to integrate the [Universal Profile Browser Extension](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl) into Web3-Onboard's connection module. In case the [Universal Profile Browser Extension](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl) is not installed, the user will be taken to its download page.
```
Comment (CJ42) [inline docs/learn/dapp-developer/web3-onboard.md:12]:
```suggestion
Our [Web3-Onboard configuration](https://github.com/lukso-network/web3-onboard-config) can be used to integrate the [Universal Profile Browser Extension](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl) into Web3-Onboard's connection module. If the [Universal Profile Browser Extension](https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl) is not installed, the user will be taken to its download page.
```

File: docs/learn/dapp-developer/web3-onboard.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+You can install Web3-Onboard on the dApp using the related [NPM package](https://www.npmjs.com/package/@lukso/web3-onboard-config)
+
+```shell
+npm i @web3-onboard/core @lukso/web3-onboard-config @web3-onboard/injected-wallets
+```
+
+## Setup
+
+After installation, you can proceed to configure your extensions that you want to dock onto your dApp.
```
Comment (CJ42) [inline docs/learn/dapp-developer/web3-onboard.md:24]: I guess?

```suggestion
After installation, you can proceed to configure your extension that you want to dock onto your dApp.
```

File: docs/learn/dapp-developer/web3-onboard.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+:::caution Refferal Links
```
Comment (CJ42) [inline docs/learn/dapp-developer/web3-onboard.md:26]:
```suggestion
:::caution Referral Links
```

File: docs/learn/dapp-developer/web3-onboard.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+Be aware that anyone implementing Web3-Onboard can modify the download link to the extension.
+
+:::
+
+:::success
+
+The Web3-Onboard configuration and calls should be set up as a global `context` or `component`, acsessable to every page or component of your application layout that is interacting with the blockchain.
```
Comment (CJ42) [inline docs/learn/dapp-developer/web3-onboard.md:34]:
```suggestion
The Web3-Onboard configuration and calls should be set up as a global `context` or `component`, accessible to every page or component of your application layout that is interacting with the blockchain.
```

**Review by CJ42 — COMMENTED** [review]
Added some review comments for typos mainly. LGTM otherwise

**Comment by Hugoo** [comment]
Would be nice to make also a link to this article from the README of this repo:

https://github.com/lukso-network/web3-onboard-config

**Review by Hugoo — APPROVED** [review]
nice

**Pull request merged**